index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>mobile-project</title>
+</head>
+<body>
+  <header>
+  <nav class="flex-nav">
+    <div>
+    <img src="images/MyLogo.png" alt="logo">
+    </div>
+    <div>
+    <img src="images/Portoflio.png" alt="Portoflio" class="right">
+    <img src="images/About.png" alt="About" class="right">
+    <img src="images/Contact.png" alt="Contact" class="right">
+  </div>
+  </nav>
+</header>
+  <main class="flex-main">
+    <section class="flex-center">
+   <h1>I’m Alexandra<br>
+    Glad to see <br> you!</h1>
+    <p class="">I’m a software developer! I can help you build a product , feature or we
+      bsite Look through some of my work and experience! If you like wha
+      t you see and have a project you need coded, don’t 
+      hestiate to contact me.</p>
+     
+    </section>
+      <section class = "flex-even">
+        <div><p>LETS CONNECT</p></div>
+        <div>
+        <img src="images/twitter.png" alt="twitter">
+        <img src="images/Vector.png" alt="Vector">
+        <img src="images/medium.svg" alt="medium">
+        <img src="images/github.png" alt="github">
+        <img src="images/angellist.png" alt="angellist">
+      </div>
```

Comment:
- [ ] `LETS CONNECT` should be a link (`a` tag) as it is an invitation to connect it makes sense for it to be a link. It also helps improve accessibility for users with visual impairments. Also, add the necessary stylings for this element.
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
`ul` -  `li` orders the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group, this helps with proper accessibility and webpage optimisation.
Kindly use `ul` - `li` for semantic correctness whenever you are trying to list items. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.
- [ ] There should be spacing between the `LETS CONNECT` and the social media links. Kindly refer to my comment above on following the provided designs on Figma.

Reply:
This issue is not addressed. Kindly make the suggested changes by the previous TSE. pray